test/e2e/performance_basic_test.go

```diff
@@ -27,17 +27,92 @@ import (
 
 var (
 	defaultResNum = 10000
-	resNumStr     = os.Getenv("PERF_RES_NUMBER")
+	// resNumStr is the number of resource rules to be created
+	// if not set, defaultResNum will be used
+	resNumStr = os.Getenv("PERF_RES_NUMBER")
```

Comment:
I think that for package variables it's better to use more elaborate names, e.g. instead of `resNumStr` I'd go for `resourcesNumberString`. Rationale: when I read a fragment of code that uses the variable, I don't have to jump to its godoc to roughly understand what it represents.